python/src/QuantileConfidence_doc.i.in

```diff
@@ -253,6 +295,42 @@ Examples
 
 // ---------------------------------------------------------------------
 
+%feature("docstring") OT::QuantileConfidence::computeBilateralConfidenceIntervalWithCoverage
+"Evaluate a bilateral confidence interval of a quantile.
+
+Refer to :func:`computeBilateralConfidenceInterval()`
+
+Parameters
+----------
+sample : 2-d sequence of float
+    Sample of the variable :math:`X`.
+
+Returns
+-------
+ci : :class:`~openturns.Interval`
+    Quantile confidence interval.
+
+coverage : float
+    The actual coverage probability of the confidence interval, i.e. the largest possible value of :math:`\beta`.
```

Comment:
Not exactly. This might not be the _largest possible value of beta_, because it depends on the meaning of this sentence.
```suggestion
    The `coverage` is the value of $P({X_{(k_1)} \leq x_{\alpha} \leq X_{(k_2)}})$.
    The coverage is guaranteed to be greater or equal to $\beta$.
```